package.json

```diff
@@ -23,10 +23,10 @@
   ],
   "scripts": {
     "size": "size-limit",
-    "test": "standard && ava",
-    "testci": "standard && ava && npm run size",
+    "testci": "standard &&  nyc ava tap test.js && npm run size",
     "prepublish": "npm run check-deps",
-    "check-deps": "nsp check"
+    "check-deps": "nsp check",
+    "report-coverage": "nyc report --reporter=lcov > coverage.lcov && codecov"
```

Comment:
=> `nyc report --reporter=text-lcov | coveralls`

Reply:
Ok here i am bit confused. I have been informed that we should be using codecov. So am just trying to know will it be codecov integration or coveralls?

Reply:
My mistake, ignore this comment.